ghcup-vanilla-0.0.8.yaml

```diff
@@ -5408,6 +5407,89 @@ ghcupDownloads:
             unknown_versioning: *cabal-31210-arm64
           Linux_UnknownLinux:
             unknown_versioning: *cabal-31210-arm64
+    3.14.1.0:
+      viTags:
+        - Latest
```

Comment:
Why is this not recommended?

Reply:
Good question, I'm not sure it makes sense in the vanilla channel until we (upstream) have more feedback on this release?

Reply:
Most releases just `Latest == Recommended`. At least that's how it's been so far. I think it kinda makes sense, but in the end it's up to the discretion of upstream. 

The main GHCup channel has its own policies.

Reply:
Ok I see